2019-08-31 14:03:25,308 ERROR: Exception on /index [GET] [in c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\routes.py", line 30, in index
    posts=posts) #user=current_user
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\templates\index.html", line 18, in template
    {% include '_post.html'} <!--{{ post.author.username }} says: <b>{{ post.body }}</b>-->
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 199, in parse_for
    body = self.parse_statements(('name:endfor', 'name:else'))
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 288, in parse_include
    node.template = self.parse_expression()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 432, in parse_expression
    return self.parse_condexpr()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 437, in parse_condexpr
    expr1 = self.parse_or()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 450, in parse_or
    left = self.parse_and()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 459, in parse_and
    left = self.parse_not()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 470, in parse_not
    return self.parse_compare()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 474, in parse_compare
    expr = self.parse_math1()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 496, in parse_math1
    left = self.parse_concat()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 507, in parse_concat
    args = [self.parse_math2()]
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 517, in parse_math2
    left = self.parse_pow()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 528, in parse_pow
    left = self.parse_unary()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 546, in parse_unary
    node = self.parse_primary()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\parser.py", line 564, in parse_primary
    next(self.stream)
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\lexer.py", line 359, in __next__
    self.current = next(self._iter)
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\lexer.py", line 562, in wrap
    for lineno, token, value in stream:
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\jinja2\lexer.py", line 683, in tokeniter
    filename)
jinja2.exceptions.TemplateSyntaxError: unexpected '}'
2019-08-31 14:04:20,341 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:42]
